Good-Bids/goodbids
# [#466] Remove Default themes.

# Summary

The title says it all, removing all the WP themes besides the `GoodBids Nonprofit` and `GoodBids Main`

## Issues

* #466

## Testing Instructions

1. Go to the main site and make sure we only have two themes `GoodBids Nonprofit` and `GoodBids Main`.


## Screenshots


<img width="1286" alt="Screenshot 2024-02-23 at 3 59 42 PM" src="https://github.com/Good-Bids/goodbids/assets/91974372/7187828f-4a0c-44d2-8726-748dac0f7199">
